pyvrp/cpp/Route.cpp

```diff
@@ -218,8 +219,9 @@ Route::Route(ProblemData const &data, Trips trips, size_t vehType)
         prizes_ += trip.prizes();
 
         auto const [x, y] = trip.centroid();
-        centroid_.first += (x * trip.size()) / size();
-        centroid_.second += (y * trip.size()) / size();
+        auto const div = std::max<size_t>(size(), 1);  // max 1 avoids zero div
+        centroid_.first += (x.get() * trip.size()) / div;
+        centroid_.second += (y.get() * trip.size()) / div;
```

Comment:
I think this was a subtle bug, because `size()` can be zero for empty routes.

Reply:
Haha, yeah, this was bugged:

<img width="784" height="311" alt="image" src="https://github.com/user-attachments/assets/ae44a016-b20f-44ca-9f13-62cba3cfbc55" />